### PR TITLE
Defaulting the line number to UID rather than null

### PR DESCRIPTION
**Pull request by manoj-fsbtech**

### CODE REVIEW

File: lib/reporter.ts [hunk 1]
```diff
@@ -306,7 +308,10 @@ export class CucumberJsJsonReporter extends WDIOReporter {
     /**
      * Add step data to the current running scenario
      */
-    public addStepData ( test: TestStats | HookStats ): void {
+    public addStepData( test: TestStats | HookStats ): void {
+        log.info( '&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&' );
```
Comment (wswebcreation) [inline lib/reporter.ts:312]: Can you remove the logs?

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Thanks for this PR and sorry for my late response. 

Can you check the comment and also make sure that this is covered in a unit test?

Thanks!

**Comment by wswebcreation** [comment]
@manoj-fsbtech 

Can you also check the unit tests, they are breaking now

**Comment by Nemesischild** [comment]
Is there any update on when this branch will be merged & made available for general use. This null line issue is blocking me on the next phase of my automation & CI/CD integration

**Comment by christian-bromann** [comment]
@Nemesischild mind raising a new PR so we can move forward with this?

**Comment by christian-bromann** [comment]
Closing due to inactivity. Please re-raise the PR if there is still interest.

**Pull request closed without merging**